css/color-function/index.md

```diff
@@ -0,0 +1,255 @@
+---
+title: "`color()`"
+description: "Обеспечиваем поддержку неочевидных цветов и преобразуем один цвет в другой."
+authors:
+  - solarrust
+contributors:
+  - skorobaeus
+keywords:
+  - функция color
+  - функция цвета
+related:
+  - css/web-colors
+  - css/color
+  - css/mix-blend-mode
+tags:
+  - article
+---
+
+Новая CSS-функция `color()` позволяет по-новому указывать значение цвета.
+
+Можно использовать как значение для любых свойств, где нужен цвет. Например, [`color`](/css/color/), [`border-color`](/css/border-color/) или [`accent-color`](/css/accent-color/).
+
+У функции два способа использования:
+
+- _абсолютное значение_ для указания цвета в цветовой области, отличной от sRGB;
+- _относительное значение_ для перевода цвета из одного цветового пространства в другое.
+
+## Абсолютное значение
+
+В CSS много способов [обозначения цвета](/css/web-colors/) в зависимости от нужного цветового пространства. Возможности работы с цветами изменяются, цветовых пространств становится всё больше. Пока не для каждого пространства существуют отдельные функции типа `rgb()` или `hsl()`, и некоторые цвета можно описать только при помощи `color()`.
+
+Например, цвета из пространства Display P3 можно указать только через эту функцию. Так записывается аналог цвета `#00bbff`:
```

Comment:
```suggestion
Например, цвета из пространства DCI-P3 (коротко P3) можно указать только через эту функцию. Так записывается аналог цвета `#00bbff`:
```

Reply:
Короче, после мучительного ресёрча я узнала, что пространство P3 — это то, что поддерживает и Chrome, и Safari. Но конкретная имплементация у Apple называется как раз Display P3. Так что здесь предлагаю использовать более универсальное название для этой штуковины.

> DCI-P3 colors are supported in CSS on Safari since 2017 (version 10.1) and Google Chrome since March 2023 (version 111) browsers.

> Display P3 is a variant developed by Apple Inc. for wide-gamut displays.

Reply:
Имхо, это никакой ментальной связи с синтаксисом не создаст. Ведь мы пишем именно `display-p3` в значении.

![CleanShot 2024-06-13 at 19 54 58@2x](https://github.com/doka-guide/content/assets/13676514/647f8104-809f-4bf3-b752-8558e5864161)

Reply:
Но это же значение, а не официальное название пространства.